library/src/test/java/com/chuckerteam/chucker/TestTransactionFactory.kt

```diff
@@ -0,0 +1,95 @@
+import com.chuckerteam.chucker.internal.data.entity.HttpTransaction
+import java.util.Date
+
+object TestTransactionFactory {
+
+    internal fun createTransaction(method: String): HttpTransaction {
+        return HttpTransaction(
+            id = 0,
+            requestDate = Date(1300000).time,
+            responseDate = Date(1300300).time,
+            tookMs = 1000L,
+            protocol = "HTTP",
+            method = method,
+            url = "http://localhost/getUsers",
+            host = "localhost",
+            path = "/getUsers",
+            scheme = "",
+            responseTlsVersion = "",
+            responseCipherSuite = "",
+            requestContentLength = 1000L,
+            requestContentType = "application/json",
+            requestHeaders = null,
+            requestBody = null,
+            isRequestBodyPlainText = true,
+            responseCode = 200,
+            responseMessage = "OK",
+            error = null,
+            responseContentLength = 1000L,
+            responseContentType = "application/json",
+            responseHeaders = null,
+            responseBody =
```

Comment:
```suggestion
            responseBody = "{"field": "value"}"
```

Should also work right?

Reply:
Hmm it should be something like that:
`responseBody = "{\"field\": \"value\"}"`
Should I fix it and add to this PR?

Reply:
Sorry I've updated my suggestion. If you use `"""` you don't need to escape all the other quotes

Reply:
Add change from your comment to my last commit and ktlint report an error:
`TestTransactionFactory.kt:31:27: Missing newline after "=" (experimental:indent)`